### PR TITLE
Merge vampires from Aurora

### PR DESCRIPTION
**Pull request by aztectornado**
Shapeshift is the only thing that seems to not work with these. Otherwise, it's pretty much the same as whatever they had. 

todo: Butcher vampires to actually be more fitting.

### CODE REVIEW

**Comment by cookays** [comment]
yay!

**Comment by staticcharged** [comment]
Awesome, thanks so much!

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 1]
```diff
@@ -93,6 +93,31 @@
...
+						return 0
+					if(mind && mind.vampire && (mind in ticker.mode.vampires))
+						H << "\red Your fangs fail to pierce [src.name]'s cold flesh"
+						return 0
+					if(get_species() == "Machine")
+						H << "\red They have no blood"
+						return 0
+					//we're good to suck the blood, blaah
+					//and leave an attack log
+					H.attack_log += text("\[[time_stamp()]\] <font color='red'>Bit [src.name] ([src.ckey]) in the neck and draining their blood</font>")
```
Comment (woothie) [inline code/modules/mob/living/carbon/human/human_attackhand.dm]: Muh grammar

**Comment by woothie** [comment]
Aside from that, and that vamps can actually pick up null rods and weaken themselves with it, we're pretty golden. And the shriek has insane range, covers the whole screen. Which is k I guess, it's up for debate. Thanks a million Patch <3

**Pull request merged**